lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -46,14 +53,18 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if share_settings:
                       <%
                        facebook_share_text = _("I completed the {course_title} course on {platform_name}.").format(course_title=accomplishment_course_title, platform_name=platform_name)
+                       twitter_share_text = _("I completed the {course_title} course on {platform_name} and here is the link to my certificate.").format(course_title=accomplishment_course_title, platform_name=platform_name)
```

Comment:
I think, we need to shorten the text length for twitter sharing. as per twitter development API
"The length of your passed Tweet text should not exceed 140 characters when combined with any passed hashtags, via, or url parameters. " @catong